src/app/ui-kit/layout/header/header.component.scss

```diff
@@ -0,0 +1,33 @@
+.nav-item-animation:before {
+  @apply rounded-lg;
+
+  background-color: theme('colors.muted');
+  bottom: -3px;
+  content: '';
+  height: 2px;
+  left: 0;
+  position: absolute;
+  transform-origin: right;
+  transition: width 0.6s ease-in-out;
+  width: 0;
+}
+
+.nav-item-animation:hover:before {
+  width: 100%;
+}
+
+.nav-item-animation:after {
+  background-color: transparent;
+  bottom: 0;
+  content: '';
+  height: 2px;
+  position: absolute;
+  right: 100%;
+  transform-origin: right;
+  transition: width 0.6s ease-in-out;
+  width: 0;
+}
+
+.nav-item-animation:hover:after {
```

Comment:
Can we use the parent selector SCSS approach for this file? Like: .nav-item-animation { &:hover {...} }. Is this possible? https://sass-lang.com/documentation/style-rules/parent-selector/

Reply:
I'll try